modules/mako.nix

```diff
@@ -0,0 +1,27 @@
+{pkgs, config, lib, ... }:
+
+with config.lib.stylix.colors.withHashtag;
+with config.stylix.fonts;
+{
+  # Referenced https://github.com/stacyharper/base16-mako
+  home-manager.sharedModules = [{
+    programs.mako = {
+      backgroundColor = base00;
+      borderColor = base0D;
```

Comment:
Do we need to set `progressColor` too?

Reply:
Probably a good idea. I forgot that option existed.